sql/updates/world/creature_template_2018_03_09_00.sql

```diff
@@ -24,3 +39,7 @@ DELETE FROM `creature_text` WHERE `entry`=72057 AND `groupid`=5 AND `id`=0;
 INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `comment`) VALUES (72057, 5, 0, 'You will feel but a fraction of my agony.', 14, 0, 100, 0, 0, 38980, 'ORDOS - SAY_POOL_OF_FIRE');
 DELETE FROM `creature_text` WHERE `entry`=72057 AND `groupid`=6 AND `id`=0;
 INSERT INTO `creature_text` (`entry`, `groupid`, `id`, `text`, `type`, `language`, `probability`, `emote`, `duration`, `sound`, `comment`) VALUES (72057, 6, 0, 'Burn!', 14, 0, 100, 0, 0, 38981, 'ORDOS - SAY_BURNING_SOUL');
+
+-- Database insert Error Fix
```

Comment:
phase mask is uint32 why changing the structure to uint8?

Reply:
```sql
SQL(p): INSERT INTO creature (guid, id , map, spawnMask, phaseMask, modelid, equipment_id, position_x, position_y, position_z, orientation, spawntimesecs, spawndist, currentwaypoint, curhealth, curmana, MovementType, npcflag, unit_flags, dynamicflags) VALUES (8189273, 69099, 1064, 1, 4294967295, 0, 0, 7137.51, 5141.57, 99.7233, 2.35221, 300, 0, 0, 12725, 0, 0, 0, 0, 0)
 [ERROR]: [1264] Out of range value for column 'phaseMask' at row 1
Unhandled MySQL errno 1264. Unexpected behaviour possible.
````

There is your reason. I tested on this way and no more error. (Just a Temp Hackfix)